go.mod

```diff
@@ -16,7 +16,6 @@ require (
 	github.com/olekukonko/tablewriter v0.0.4
 	github.com/onsi/ginkgo v1.14.2
 	github.com/onsi/gomega v1.10.3
-	github.com/otiai10/copy v1.2.0
```

Comment:
I would have expected more requirements to go away with this cleanup. `github.com/tektoncd/pipeline` and `code.gitea.io/sdk/gitea` just to name two.

Reply:
tekton is gone, gitea will be next